cmd/kar-controllers/app/options/options.go

```diff
@@ -35,6 +35,8 @@ import (
 	"os"
 	"strconv"
 	"strings"
+
+	klog "k8s.io/klog/v2"
```

Comment:
When compiling, I got:: `cmd/kar-controllers/app/options/options.go:39:2: "k8s.io/klog/v2" imported as klog and not used`